docs/getting-started-linter.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Linter Integration
+---
+
+When writing code, most developers like to see where are problems and code smells directly
+in their code, without switching to an external window or panel.
+
+If you have a **linter plugin** ([linter](https://atom.io/packages/linter) or
```

Comment:
*the linter package

(it's kinda confusing, really, because Atom has a bunch of packages that do linting, some of them called linter-xxx, some of them xxx-ide, and not all plugins with similar names are related)

Also, Atom add-ons are called packages, not plugins.

Reply:
It's confusing and that's why I give these links. it makes clear what kind of packages I am talkg about.